packages/theme/src/types.ts

```diff
@@ -1,5 +1,7 @@
 import { themeDefault } from './themes'
 
+export type ThemeName = 'light' | 'dark'
```

Comment:
One more ThemeName? We have two above, why do we need another one?

Reply:
outdated